### DISCUSSION TITLE
allow the skin editor for mania to resize the width of the playing area

### DISCUSSION THREAD

**Discussion by Mast3rwaf1z** [ppy/osu#19567] [Ideas] [answered]
Hi! I was trying out Lazer again today where I wanted to try out the touch controls. I tested out mania and noticed that the 4 key area is way too small for touch controls so I think adding the ability to change that in the skin editor would be a good addition

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/19247 / https://github.com/ppy/osu/discussions/16783
